bin/runtime/src/lib.rs

```diff
@@ -383,12 +391,82 @@ impl<I: From<u32>> ::frame_support::traits::Get<I> for SessionPeriod {
 impl pallet_session::Config for Runtime {
     type Event = Event;
     type ValidatorId = <Self as frame_system::Config>::AccountId;
-    type ValidatorIdOf = ConvertInto;
+    type ValidatorIdOf = pallet_staking::StashOf<Self>;
     type ShouldEndSession = pallet_session::PeriodicSessions<SessionPeriod, Offset>;
     type NextSessionRotation = pallet_session::PeriodicSessions<SessionPeriod, Offset>;
-    type SessionManager = pallet_aleph::AlephSessionManager<Self>;
+    type SessionManager = pallet_session::historical::NoteHistoricalRoot<Self, Staking>;
```

Comment:
What does this change mean to us?

Reply:
That now pallet staking is responsible for triggering new sessions and not pallet aleph. Pallet historical is just a wrapper that stores previous data.

Reply:
So historical wraps `Staking`? Why do we need that?

Reply:
This is used for equivocations in block production and finality. It provides a basic implementation of https://paritytech.github.io/substrate/master/frame_support/traits/trait.KeyOwnerProofSystem.html which is then extended in those modules.

Reply:
Ok, but is this functionality used in the current code, or only in the future?